### ISSUE TITLE
Easy installation as debian package

### ISSUE THREAD

**Issue by kravemir** [editorconfig/editorconfig-gedit#16]
Currently, editorconfig-gedit isn't present as debian package: https://packages.debian.org/search?searchon=names&keywords=editorconfig (and, search for gedit)

It would be great to make debian package for editorconfig-gedit, as it eases installation on Debian/Ubuntu based distributions.

**Comment by xuhdev** [comment]
Thanks for your suggestion, but Debian packages are not created from its software developers. They are created by people from Debian. To request such a package, please go for [Debian RFP](https://wiki.debian.org/RFP).

**Comment by kravemir** [comment]
Well, you don't seem thankful. You've immediately closed the issue, and didn't wait for reaction, where both sides consider discussion about issue as closed (and, there are constructive arguments).

This bad attitude towards users. It gives impression, that you don't care whether users will have a conformable way to use your project.

**Comment by kravemir** [comment]
> but Debian packages are not created from its software developers.

Wrong. Debian packages are created either by people from Debian, or by project's developers. You're able to make a Debian package for your project.

**Comment by xuhdev** [comment]
> Wrong. Debian packages are created either by people from Debian, or by project's developers. You're able to make a Debian package for your project.

I didn't see how. Reference?

**Comment by kravemir** [comment]
You can use [debian mentors](https://mentors.debian.net/), only debian developers can upload packages, but they can do it for non-official debian developers (ie. project package maintainers):

> Only approved members of the Debian project (Debian Developers) are granted the permission to upload software packages into the Debian distribution. **Still a large number of packages is maintained by non-official developers.** How do they get their work into Debian when they are not allowed to upload their own packages directly? By means of a process called sponsorship. **Sponsorship means that a Debian Developer uploads the package on behalf of the actual maintainer.** The Debian Developer will also check the package for technical correctness and help the maintainer to improve the package if necessary. Therefore the sponsor is sometimes also called a mentor.

See also [DebianMentors FAQ](https://wiki.debian.org/DebianMentorsFaq).

If you do a good job, then you can become [Debian Maintainer](https://wiki.debian.org/DebianMaintainer) for your package:

> Debian Maintainers (DMs) are people who have a restricted ability to upload packages to the Debian archive. They can maintain packages without a sponsor. ...

From [Maintainers](https://wiki.debian.org/Maintainers):

> Debian Maintainer is an official status of people, who are allowed to upload specific packages. They don't need to request for sponsorship for these packages.
